### ISSUE TITLE
Slider 6.0.0.alpha feedback

### ISSUE THREAD

**Issue by ichim-david** [kitconcept/volto-slider-block#38]
Things that I would like to see done as improvements:
 I would like some options in the block schema such as:
- [ ] auto-play toggle
- [ ] Show or hide arrows

A11y issues:
- [x] There is no visual indication when you are focused on the previous or next button
- [x] The previous and next buttons should say next slide, previous slide not just previous button group
- [ ] When used with the light theme you have the gray bg color that can be applied in which case the non active dots are hidden

**Comment by tisto** [comment]
@ichim-david thanks for reporting and giving feedback. The a11y issues should be fixed asap. Regarding the features I'd like to have a minimal release at first. Then we have to discuss the features. As our design philosophy, we want to allow integrators to make decisions but not editors. Allowing editors to choose "show/hide arrows" for instance will lead to an inconsistent display of sliders on the site, which is something we try to avoid. The UI designer of a project should make the decision what's the "right" way of doing this, not editors (who might not know enough about UI or a11y).

The auto-play toogle is something that I would have to check. Isn't auto-play itself an a11y violation?

As said, I am ok with adding those features for integrators to enable or disable via json configuration and then giving editors the choice. Though, for kitconcept projects and for Volto-Light-Theme default, we are aiming for a good user experience by not showing too many options.

We also have to take into consideration that each option we add needs to be fully tested.

**Comment by ichim-david** [comment]
> @ichim-david thanks for reporting and giving feedback. The a11y issues should be fixed asap. Regarding the features I'd like to have a minimal release at first. Then we have to discuss the features. As our design philosophy, we want to allow integrators to make decisions but not editors. Allowing editors to choose "show/hide arrows" for instance will lead to an inconsistent display of sliders on the site, which is something we try to avoid. The UI designer of a project should make the decision what's the "right" way of doing this, not editors (who might not know enough about UI or a11y).
> 
> The auto-play toogle is something that I would have to check. Isn't auto-play itself an a11y violation?
> 
> As said, I am ok with adding those features for integrators to enable or disable via json configuration and then giving editors the choice. Though, for kitconcept projects and for Volto-Light-Theme default, we are aiming for a good user experience by not showing too many options.
> 
> We also have to take into consideration that each option we add needs to be fully tested.
@tisto you make good points, you do want to make choices for the editor but ideally you also want to make it somewhat
easy for other developers to enable or disable some options without having todo overrides.

I am thinking of a world where you create a page layout and you add the slider and you lock the editing of options
but you configure it then, do you want the arrows or not?
This way the choice is easily done TTW and not something that can be changed afterwards.

Nevertheless, these remarks are all just feedback, feel free todo whatever you want with it.

**Comment by sneridagh** [comment]
@ichim-david #39
